apps-we-use/workable.md

```diff
@@ -0,0 +1,75 @@
+## Hiring process
+
+### Overview
+
+We strive to have a bureaucracy- and ego- free hiring process that provides a welcoming experience for new Niteans. It's impossible to thoroughly test someone's fit with interviews and phone calls, so we rather do a couple of basic tests and then take them on for a [trial period].
+
+That should be your mindset when conducting interviews: you are not searching for an ideal candidate. You are on the lookout for big "no-no"'s. If someone's lacking some skills, they can be learned. If someone is used to different type of tools or processes, they can be changed. Look for problem solving prowess, ability to communicate what went wrong and ownership of mistakes.
+
+### Hiring workflow
+
+Workable guides us through the following stages of hiring a new Nitean:
+
+1. Sourcing
+   We post the job ads to various free and paid options. This is decided on a per-position basis, but here's the general idea:
+   * Workable posts to 15+ US-based free boards automatically (Glassdoor, Indeed, etc.)
+   * We post manually to Slovenian boards: slo-tech.com, internetmojster.com, facebook.com/groups/sluzbezadeveloperje
+   * We post to twitter.com/teamniteo and facebook.com/niteo.
+   * Sometimes we make an add campaign on Facebook or buy ads on StackOverflow.
+
+2. Application
+   Interested candidates get to our Application Form, hosted by Workable. The form needs to be short, with only general questions. Advanced (technical) questions to asses basic skills for the position will be done in the next step.
+
+   Members of the hiring team +1/-1 candidates based on application form answers. Whoever gets two or more +1s gets transitioned to the next step: Questionnaire. Use the `Advanced Questionnair - <position>` email template to send the questions/tasks to the candidate.
+
+3. Questionnaire
+   When the answers arrive we rate the candidate using the scorecard that is prepared in advance, and matches the `Advanced Questionnaire - <poisition>` template. Discussion happens in comments inside Workable, so everything is kept in one place.
+
+   Whoever gets two or more +1s gets transitioned to the next step: Phone Screen.
+
+4. Phone Screen
+   This is a quick chat over Skype/Hangouts/etc. with @marbeann. The goal is to confirm the candidate is able to communicate via video conferencing. Is their connection OK? Is their English OK? Can they hold a discussion on a subject?
+
+   @marbe also confirms the following:
+   * weekly commitment for the trial period (full- or part-time),
+   * when can they start,
+   * expected ballpark salary (more detailed salary discussion in next step).
+
+5. Interview
+   Before the interview we need to select the Nitean who will mentor the candidate should the candidate start the trial period.
+
+   Then the proposed mentor for the candidate and at least one of founders (partners?), along with anyone who is interested and/or who will work with the candidate, get on Zoom to have a chat with the candidate.
```

Comment:
`one of the Partners`

Reply:
I specifically went for Founders as Partners does not mean much to people outside of Niteo. But for any candidate it always feels like one matters if one of founders is part of the interview.